README.md

```diff
@@ -12,14 +14,57 @@ Assuming you already have [Docker](https://www.docker.com/) installed, you can s
 
 ...and view the site locally at: http://localhost:8036/
 
-## Running tests
+## Developing the dashboard
+
+To start development with the dashboard follow the [Getting started](#geting-started) instructions and any changes to the code will be auto built and the browser refreshed.
 
-To run tests locally:
+### To run the tests.
 
 ```
 ./run test
 ```
 
+### Generating a release for Juju
+
+```
+./run exec yarn run generate-release-tarball
+```
+
+### Accessing a lxd bootstrapped Dashboard that's also within a `multipass` VM
+
+When you want to access a dashboard that was automatically installed with a `juju bootstrap` within an lxd in a `multipass` vm from the host.
+
+- [Create a ssh tunnel](#creating-a-tunnel-from-an-lxd-controller-to-a-multipass-host)
+- In the host
+  - `multipass info dev | grep IPv4` and take note of the IP address.
+  - In your browser visit `http://<that ip>:17070`
+  - Log in using the credentials provided from the `juju dashboard` command.
+
+### Accessing a local bootstrapped controller from a hosted Dashboard.
+
+When you have a Juju controller bootstrapped in an lxd within a `multipass` vm and a dashboard being hosted from the same `multipass` vm but accessed from the host.
+
+- [Create a ssh tunnel](#creating-a-tunnel-from-an-lxd-controller-to-a-multipass-host)
+- In the host
+  - `multipass info dev | grep IPv4` and take note of the IP address.
+  - Open `config.js` and modify the following values:
+    - 'baseControllerURL`should be output from the`multipass info`call above with the port`17070`.
+    - `identityProviderAvailable` to `false`.
+    - `isJuju` to `true`.
+
+### Creating a tunnel from an lxd controller to a `multipass` host.
+
+If you're bootstrapping a Juju controller within a `multipass` VM you will not have access to the dashboard from the host because there is no network path to it. To create a tunnel to access the dashboard follow these steps.
+
+- After bootstrapping a controller on `lxd` in your `multipass` VM...
+- In the multipass vm
+  - `juju switch controller && juju ssh 0`
+- In the lxd container
```

Comment:
How do I access the LXD container from my Multipass?

Reply:
the `juju ssh 0` in the above command will ssh you into the lxc that Juju created for it's "machine 0"